application/repositories.go

```diff
@@ -1,11 +1,27 @@
-package remoteworkitem
```

Comment:
I'm assuming due to package cycle, I never touched TrackerRepository in my example, only the upload method. .